src/main/kotlin/ch/hevs/cloudio/cloud/apiutils/EndpointManagementUtil.kt

```diff
@@ -116,10 +120,22 @@ object EndpointManagementUtil {
         else {
             //send message to amq.topic queue
 
-            // TODO: Detect actual serialization format from endpoint data model.
+            val endpointUUID = attributeSetRequest.attributeTopic.split("/")[0]
+            val endpoint = endpointEntityRepository.findById(UUID.fromString(endpointUUID)).get()
+            val serializedAttribute =
+                if(endpoint.endpoint.supportedFormats.contains("JSON")){
+                    println("JSON")
```

Comment:
The evil println adder has hit again ;-)

Reply:
It keeps trying to enter my code, it's an everyday battle x)